tests/framework-test/tutorial_fsi_3d.head

```diff
@@ -222,7 +222,7 @@ MATERIALTANGENT                 analytical
 //
 // Application of nonlinear inertia terms
 //     No,no,Standard,standard,Rotations,rotations
-MASSLIN                         No
+MASSLIN                         None
```

Comment:
... to this. This should be the error. Anyways, the error can be better here. Fix coming up.

Reply:
Thanks for the helping hand! I will fix the capitalization.